components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-list-load.jsp

```diff
@@ -50,7 +51,7 @@
     }
 %>
 <script type="text/javascript">
-    location.href = "<%=callback%>";
+    location.href = "<%=Encode.forJavaScriptBlock(callback)%>";
```

Comment:
callback holds a trusted string which is hard coded and does not contain special characters. Thus, no need to encode here